hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/TargetTagRepository.java

```diff
@@ -81,4 +81,9 @@
     @Transactional
     @Query("DELETE FROM JpaTargetTag t WHERE t.tenant = :tenant")
     void deleteByTenant(@Param("tenant") String tenant);
+
+    @Override
+    // Workaround for https://bugs.eclipse.org/bugs/show_bug.cgi?id=349477
```

Comment:
Is this still relevant? The bug states to be fixed in 2.7.2. HawkBit is already using 2.7.3

Reply:
First of all, thanks for pointing this out! The workaround mentioned above is also part of other *Repository classes (see TargetRepository, DistributionSetRepository, etc.) so this is not really relevant for the Vaadin Migration. However, I have taken a look and it seems that the bug still exists, I do get an invalid SQL syntax exception for different database engines (tested with H2 and MSSQL) due to additional round brackets surrounding the IN arguments:
* ... WHERE ((id IN ((?,?))) AND (tenant = ?)) -> does NOT work (NO workaround applied)
* ... WHERE ((id IN (?,?)) AND (tenant = ?))  -> does work (workaround applied)

Reply:
I see, thanks for verification that the issue still exists!